packages/eas-cli/src/build/utils/resourceClass.ts

```diff
@@ -57,7 +57,7 @@ export async function resolveBuildResourceClassAsync<T extends Platform>(
   const selectedResourceClass = resourceClassFlag ?? profileResourceClass;
```

Comment:
(Noticed on the screenshot:) I cannot add a comment to line 53 but can you please change it to:

```
`Build profile specifies the "${profileResourceClass}" resource class but you passed "${resourceClassFlag}" to --resource-class.\nUsing "${resourceClassFlag}" as the override.`
```
(redundant "the" and space)